Makefile

```diff
@@ -249,8 +245,14 @@ docker-push:
 	@$(eval OC ?= $(shell which istiooc 2>/dev/null || which oc))
 	@${OC} get project ${NAMESPACE} > /dev/null
 
+.openshift-find-addons: .openshift-validate
+	@$(eval JAEGER_URL ?= $(shell echo http://$$(${OC} get svc tracing -n istio-system -o jsonpath='{.spec.clusterIP}'):80))
```

Comment:
This should also check if the route is https, like what I did here: https://github.com/kiali/kiali/pull/768/files#diff-5e6fb42f8bf476d6c792f7dcceabf189R147